### PR TITLE
Done

### PR DESCRIPTION
**Pull request by JuanGongora**

### CODE REVIEW

**Comment by bhollan** [comment]
Hi @JuanGongora !

Looking over your changes here, it looks like you have an extra opening `div` tag in your edits of `index.html`.
If you can take care of that, I think we can merge your changes in.

As always, feel free to Ask A Question if you need any help getting things squared away.
CC: @aturkewi

**Comment by JuanGongora** [comment]
Hi Brian,

I fixed the error. Thanks for pointing it out.

Best,
Juan

File: index.html [hunk 1]
```diff
@@ -26,6 +26,18 @@
...
+                <h3 class="view-profile-text">View Profile</h3>
+            </div>
+            <div class="card-text-container">
+                <h4 class="student-name">Juan Gongora</h4>
+                <p class="student-location">Amherst, MA</p>
+            </div>
+        </a>
+    </div>
+
         <div class="roster-cards-container">
```
Comment (aturkewi) [inline index.html]: All the html that your adding should be _inside_ of the `<div class="roster-cards-container">` just like all the other cards.

File: index.html [hunk 2]
```diff
@@ -26,6 +26,18 @@
 
 
 
+    <div class="student-card" id="juan-gongora-card">
```
Comment (aturkewi) [inline index.html]: You need to make sure that the indentation of your card matches the same indentation of all the other student cards.

**Comment by JuanGongora** [comment]
Oops sorry about that Avidor. I fixed it now.

Regards,
Juan

**Comment by aturkewi** [comment]
Hello! We recently made a change to this repo that caused all open PRs to be close and we are not able to re-open them. I'm very sorry about the inconvenience, but I just wanted to let you know why this PR was close and that we still welcome your profile on the site! So if you're still interested in having your profile up, please feel free to open another PR.

**Pull request closed without merging**